src/sinks/elasticsearch/common.rs

```diff
@@ -145,7 +145,10 @@ impl ElasticsearchCommon {
                     )
                     .await
                     {
-                        Ok(version) => version,
+                        Ok(version) => {
+                            debug!(message = "Auto-detected Elasticsearch API version:", %version);
```

Comment:
👍 good call! I think it'll be caught by a check we have on log messages.

```suggestion
                            debug!(message = "Auto-detected Elasticsearch API version.", %version);
```